Fix BeginRenderPass links

john-h-k opened the pull request.

john-h-k committed:
Fix BeginRenderPass links

PRMerger15 commented:
@john-h-k : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims reviewed (changes requested):
It's important not to include the language tag in urls (for the benefit of any customer not using a browser set to en-us). So, for example it's important not to include the /en-us/ in https://docs.microsoft.com/en-us/windows/win32/api/d3d12/ne-d3d12-d3d12_render_pass_beginning_access_type.

After that, it's also a good idea to transform any docs.microsoft.com url into a site-relative one. That's so that the url works not just online but also in the Microsoft Help Viewer. That's easily done by taking a pass replacing all https://docs.microsoft.com/ with /. The result of this step and the previous one is to be left with urls of the form /windows/win32/api/d3d12/ne-d3d12-d3d12_render_pass_beginning_access_type. Those are site-relative urls.

Thanks!

john-h-k committed:
Apply requested changes

PRMerger14 commented:
@john-h-k : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Just another couple tweaks

For any topic in the API ref repo, any H1 is ignored by the build system (even though most topics here contain one). So it's clearer to delete them whenever you see them. Also changed any /windows/desktop/ to /windows/win32/.

stevewhims reviewed (approved):
Excellent, thank you!
